src/Controller/CollaborativeEditorsController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Drupal\collaborative_editors\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+
+/**
+ * Defines CollaborativeEditorsController class.
+ */
+class CollaborativeEditorsController extends ControllerBase {
+
+  /**
+   * Display the markup.
+   *
+   * @return array
+   *   Return markup array.
+   */
+  public function plugins() {
```

Comment:
Show the plugins under this page like the Etherpad plugin else user won't be able to access the configuration page of the plugin.